setup.py

```diff
@@ -36,6 +37,7 @@ def read_file(filename, encoding='utf8'):
     'pyasn1',  # urllib3 InsecurePlatformWarning (#304)
     'pycrypto',
     'PyOpenSSL',
+    'pyparsing>=1.5.5',
```

Comment:
is the version pin necessary? if so, please include inline comment, just like in case of other pins